torchtext/experimental/datasets/raw/sequence_tagging.py

```diff
@@ -39,8 +39,11 @@ def _construct_filepath(paths, file_suffix):
     return None
 
 
-def _setup_datasets(dataset_name, separator, root=".data"):
-
+def _setup_datasets(dataset_name, separator, root=".data", data_select=('train', 'valid', 'test')):
+    if isinstance(data_select, str):
```

Comment:
Let's create a utility for this (maybe something like [common.py](https://github.com/pytorch/text/pull/1000/files#diff-b68328742fe41f00ad1ff63e75478f7d)).

Reply:
Done

Reply:
Why not use check_default_set?